tests/components/switchbot_cloud/test_config_flow.py

```diff
@@ -0,0 +1,92 @@
+"""Test the SwitchBot via API config flow."""
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.switchbot_cloud.config_flow import (
+    CannotConnect,
+    InvalidAuth,
+)
+from homeassistant.components.switchbot_cloud.const import DOMAIN, ENTRY_TITLE
+from homeassistant.const import CONF_API_KEY, CONF_API_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Comment:
Why do we need this?

Reply:
We do not need that ! Don't even remember why I wrote this